catalogs/catalog-lakehouse-paimon/build.gradle.kts

```diff
@@ -19,6 +24,7 @@ dependencies {
     exclude("javax.servlet")
   }
   implementation(libs.commons.lang3)
+  implementation(libs.bundles.log4j)
```

Comment:
please place  `implementation(libs.bundles.log4j)` before `implementation(libs.commons.lang3)` for orders

Reply:
updated.